public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -77,6 +77,7 @@ export class AppChromeService {
     if (newState.sectionNav !== current.sectionNav || newState.pageNav !== current.pageNav) {
       if (
         newState.actions === current.actions &&
+        newState.layout === current.layout &&
```

Comment:
this is the bugfix part that fixes the issue of loading an app plugin page with layout canvas as the first page